src/test/java/org/openmrs/performance/Constants.java

```diff
@@ -33,6 +33,11 @@ public class Constants {
 	public static final String DAYS = "1072AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA";
 	public static final String ORDER = "39da3525-afe4-45ff-8977-c53b7b359158";
 
+	public static final String CONDITION_UUID = "114403AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA";
```

Comment:
```suggestion
	public static final String BACK_PAIN = "48035AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA";
```

Let's name it with the name of the condition. Also, I included the UUID of the back pain concept, just to make it a bit better.